frontend/packages/helm-plugin/src/components/list-page/HelmTabbedPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button } from '@patternfly/react-core';
+import { Trans, useTranslation } from 'react-i18next';
+import { RouteComponentProps } from 'react-router-dom';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import CreateProjectListPage from '@console/dev-console/src/components/projects/CreateProjectListPage';
+import { useAccessReview } from '@console/dynamic-plugin-sdk/src';
+import { withStartGuide } from '@console/internal/components/start-guide';
+import { Page } from '@console/internal/components/utils';
+import { MenuActions, MultiTabListPage } from '@console/shared';
+import { HelmChartRepositoryModel, ProjectHelmChartRepositoryModel } from '../../models';
+import HelmReleaseList from './HelmReleaseList';
+import HelmReleaseListPage from './HelmReleaseListPage';
+import RepositoriesPage from './RepositoriesListPage';
+
+type HelmTabbedPageProps = RouteComponentProps<{ ns: string }>;
+
+export const PageContents: React.FC<HelmTabbedPageProps> = (props) => {
+  const { t } = useTranslation();
+  const {
+    match: {
+      params: { ns: namespace },
+    },
+  } = props;
+  const [showTitle, canCreate] = [false, false];
+  const [projectHelmChartCreateAccess, loadingCreatePHCR] = useAccessReview({
+    group: ProjectHelmChartRepositoryModel.apiGroup,
+    resource: ProjectHelmChartRepositoryModel.plural,
+    verb: 'create',
+    namespace,
+  });
+  const [helmChartCreateAccess, loadingCreateHCR] = useAccessReview({
+    group: HelmChartRepositoryModel.apiGroup,
+    resource: HelmChartRepositoryModel.plural,
+    verb: 'create',
+  });
+  const [projectHelmChartEditAccess, loadingEditPHCR] = useAccessReview({
+    group: ProjectHelmChartRepositoryModel.apiGroup,
+    resource: ProjectHelmChartRepositoryModel.plural,
+    verb: 'update',
+    namespace,
+  });
+  const [helmChartEditAccess, loadingEditHCR] = useAccessReview({
+    group: HelmChartRepositoryModel.apiGroup,
+    resource: HelmChartRepositoryModel.plural,
+    verb: 'update',
+  });
+
+  const menuActions: MenuActions = {
+    helmRelease: {
+      label: t('helm-plugin~Helm Release'),
+      onSelection: () => `/catalog/ns/${namespace}?catalogType=HelmChart`,
+    },
+    projectHelmChartRepository: {
+      label:
+        projectHelmChartCreateAccess || helmChartCreateAccess ? t('helm-plugin~Repository') : null,
+      onSelection: () => `/ns/${namespace}/helmchartrepositories/~new?actionOrigin=list`,
+    },
+  };
+
+  const pages: Page[] = [
+    {
+      href: '',
+      name: t('helm-plugin~Helm Releases'),
+      component: HelmReleaseList,
+    },
+    {
+      href: 'repositories',
+      name: t('helm-plugin~Repositories'),
+      component: RepositoriesPage,
+      pageData: {
+        showTitle,
+        canCreate,
+      },
+    },
+  ];
+
+  return namespace ? (
+    projectHelmChartCreateAccess ||
+    projectHelmChartEditAccess ||
+    helmChartCreateAccess ||
+    helmChartEditAccess ? (
```

Comment:
you might want to check both edit and create accesses for showing the Repositories tab

```suggestion
    (projectHelmChartCreateAccess &&
    projectHelmChartEditAccess )||
    (helmChartCreateAccess &&
    helmChartEditAccess) ? (
```

I created this Role:
```
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: example
  namespace: deb-test
  uid: 1efc50f1-bc8a-4129-86fc-638155839557
  resourceVersion: '699005'
  creationTimestamp: '2022-08-01T17:30:20Z'
  managedFields:
    - manager: Mozilla
      operation: Update
      apiVersion: rbac.authorization.k8s.io/v1
      time: '2022-08-01T17:30:20Z'
      fieldsType: FieldsV1
      fieldsV1:
        'f:rules': {}
rules:
  - verbs:
      - get
      - watch
      - list
      - create
      - delete
    apiGroups:
      - helm.openshift.io
    resources:
      - projecthelmchartrepositories
```
and saw the restricted error in the repositories tab because the user didn't have access to update the chart repository